int32 agent3_value

uint16 agent3_posx
uint16 agent3_posy
uint16 agent3_post

uint16 agent3_ballx
uint16 agent3_bally
uint16 agent3_ball_detected

uint8 agent3_pass
uint8 agent3_condition